### PR TITLE
fix(#7284): log print ex details.

### LINKED ISSUE #7284: [1.4.2] Bug: some place log do not print ex details.
**Describe the bug**
some place log do not print ex details. like this:
```java
Loggers.SRV_LOG.error("[IP-DEAD] failed to delete ip automatically, ip: {}, error: {}", instance.toJson(),  throwable);
```

### PR DESCRIPTION
**Pull request by CherishCai**
fix #7284

### CODE REVIEW

**Review by KomachiSion — CHANGES_REQUESTED** [review]
Almost these log is async task, which will make the error log print many lines. We should discuss it first.

File: naming/src/main/java/com/alibaba/nacos/naming/core/ServiceManager.java [hunk 1]
```diff
@@ -298,7 +298,7 @@ public void run() {
                 updatedHealthStatus(namespaceId, serviceName, serverIP);
             } catch (Exception e) {
                 Loggers.SRV_LOG
-                        .warn("[DOMAIN-UPDATER] Exception while update service: {} from {}, error: {}", serviceName,
```
Comment (realJackSun) [inline naming/src/main/java/com/alibaba/nacos/naming/core/ServiceManager.java:301]: Why to remove {}?

**Review by realJackSun — CHANGES_REQUESTED** [review]

File: naming/src/main/java/com/alibaba/nacos/naming/core/ServiceManager.java (hunk 1 above)
Comment (CherishCai) [reply naming/src/main/java/com/alibaba/nacos/naming/core/ServiceManager.java:301]: placeholder {} let e.toString(), will lost error stackTrace details.

**Comment by CherishCai** [comment]
> Almost these log is async task, which will make the error log print many lines. We should discuss it first.

Yes, Nacos should use the slf4j facade api for logging. Should not recommend ExceptionUtil.getStackTrace()

**Comment by CherishCai** [comment]
I'm sorry it's late to sign the CLA.

**Review by KomachiSion — CHANGES_REQUESTED** [review]
Whether the original author want to log message only?

**Comment by CherishCai** [comment]
> Whether the original author want to log message only?

If you want to know the details of the error, I think it needs to be handled in this way. Maybe the original author didn't notice it.

**Review by KomachiSion — CHANGES_REQUESTED** [review]
The PushService will print so many error logs, because the push will always retry for failed tasks. should we only print the message not all stack?

**Comment by CherishCai** [comment]
> The PushService will print so many error logs, because the push will always retry for failed tasks. should we only print the message not all stack?

Okay, I'll go back and look at it tonight and change back to the old logic

**Review by KomachiSion — APPROVED** [review]

**Pull request merged**